nomos-da/kzgrs-backend/src/encoder.rs

```diff
@@ -409,7 +409,7 @@ pub mod test {
     }
 
     #[test]
-    #[should_panic(expected = "Encoding only zeroes is not allowed in Nomos DA network.")]
+    #[should_panic(expected = "assertion failed: polynomial_degree.is_power_of_two()")]
```

Comment:
why is this in this PR? or this change comes from a merge?

Reply:
when I merged master it breaked clippy warning for no message on panic

Reply:
But this shouldn't be fixed here. You should have open a pr on master, then re-rebase this branch if its crucial.